src/app.test.ts

```diff
@@ -34,15 +33,15 @@ describe('Express app (without Arranger)', () => {
     const getProject = (_s: string) => ({} as ArrangerProject);
 
     beforeEach(() => {
-        const sqs = new AWS.SQS({ apiVersion: '2012-11-05' });
+        const sqs = new SQSClient({});
```

Comment:
no need to use apiVersion?

Reply:
https://docs.aws.amazon.com/sdk-for-javascript/v3/developer-guide/javascript_sqs_code_examples.html

Not in this exemple, but i can't test it localy